sources/packages/api/src/services/education-program-offering/education-program-offering.service.ts

```diff
@@ -25,35 +28,13 @@ export class EducationProgramOfferingService extends RecordDataModelService<Educ
   async createEducationProgramOffering(
     locationId: number,
     programId: number,
-    educationProgramOffering: CreateEducationProgramOfferingDto,
+    educationProgramOffering: SaveEducationProgramOfferingDto,
```

Comment:
We should not save directly the DTO received from the payload in the controller. In this way, if the API receives any other field that is not supposed to be saved it will be persisted. It basically opens the entire table to be updated without any control from the API. Please revert it as it was before.

Reply:
This was creating code duplication error in Sonar check. Thats why created a method. Is that ok?